functions/src/models/requests/index.ts

```diff
@@ -300,22 +312,14 @@ export const RequestFirestoreConverter: FirestoreDataConverter<Request> = {
     return Request.factory(data.data());
   },
   toFirestore: (modelObject: Request): DocumentData => {
-    return {
-      cavUserRef: modelObject.cavUserRef,
-      cavUserSnapshot: modelObject.cavUserSnapshot ? modelObject.cavUserSnapshot.toObject() : null,
-      pinUserRef: modelObject.pinUserRef,
-      pinUserSnapshot: modelObject.pinUserSnapshot.toObject(),
-      title: modelObject.title,
-      description: modelObject.description,
-      latLng: JSON.stringify(modelObject.latLng),
-      streetAddress: modelObject.streetAddress,
-      status: modelObject.status,
-      createdAt: modelObject.createdAt.toDate(),
-      updatedAt: modelObject.updatedAt.toDate(),
-      pinRating: modelObject.pinRating,
-      cavRating: modelObject.cavRating,
-      pinRatedAt: modelObject.pinRatedAt?.toDate() || null,
-      cavRatedAt: modelObject.cavRatedAt?.toDate() || null,
-    };
+    const object = modelObject.toObject();
```

Comment:
have you tried running the `tests` with these changes? they should be throwing errors due to the Timestamp.
but it could very well be an issue with the types